src/Rings/mpoly-ideals.jl

```diff
@@ -326,15 +326,27 @@ end
 #######################################################
 
 @doc Markdown.doc"""
-    :in(f::MPolyElem, I::MPolyIdeal)
+    ideal_membership(f::MPolyElem, I::MPolyIdeal)
 
-Returns `true` if `f` is contained in `J` and `false`, otherwise.
+Returns `true` if `f` is contained in `I`, `false` otherwise. Alternatively, use `f in I`. 
 """
-function Base.:in(f::MPolyElem, I::MPolyIdeal)
+function ideal_membership(f::MPolyElem, I::MPolyIdeal)
   groebner_assure(I)
   Sx = base_ring(I.gb.S)
   return Singular.iszero(reduce(Sx(f), I.gb.S))
 end
+Base.:in(f::MPolyElem, I::MPolyIdeal) = ideal_membership(f,I)
+#######################################################
+@doc Markdown.doc"""
+    radical_membership(f::MPolyElem, I::MPolyIdeal)
+   
+Returns `true` if `f` is contained in the radical of `I`, `false` otherwise.
```

Comment:
As suggested elsewhere, how about putting the examples into the docstrings, so that REPL users also see them in the help? I.e.:
````suggestion
Returns `true` if `f` is contained in the radical of `I`, `false` otherwise.

```@repl oscar
R, (x,) = PolynomialRing(QQ, ["x"])
f = x
I = ideal(R,  [x^2])
radical_membership(f, I)
```
````

(IMHO even better would be to turn these into `jldoctest`, but I don't want to suggest too many things at once ;-))

Reply:
Christian wollte da ab und zu drüber gehen und das machen. Ich schaue mir das aber auch selbst an für die Zukunft.